### DISCUSSION TITLE
I wonder if the style isn't compatible with themes with images or cause of my bad apply of the theme.

### DISCUSSION THREAD

**Discussion by Godiesc** [black7375/Firefox-UI-Fix#558] [Q&A]
The selected tab shows a color in the selected tab instead to display the image behind. 

![imagen](https://user-images.githubusercontent.com/22057609/210278511-67bdd8e4-4424-4e3b-8f6e-775b68bd0d29.png)

**Comment by black7375** [comment]
What's your theme?
I'll check it out tomorrow

**Comment by Godiesc** [comment]
I installed the Lepton version 7, and additionally I used this: https://github.com/black7375/Firefox-UI-Fix/wiki/Show-Off-Your-Config#australis-like-theme  but with a theme with images like this: https://addons.mozilla.org/es/firefox/addon/beautiful-gx-fantasy/

**Reply by black7375** [reply]
I found the cause.
This is because of `drop-shadow` in the `filter`.
https://github.com/black7375/Firefox-UI-Fix/blob/f3f0c6472658a8a58928f2a3b75c05c9992424aa/src/tab/selected_tab/_box_shadow.scss#L54

**Reply by Godiesc** [reply]
I should delete that line?
I just have a leptonChrome.css with all the rules, I should download with code-> download ?.... I'm lost jeje

![imagen](https://user-images.githubusercontent.com/22057609/210426454-6ab16906-ac67-4406-a0d7-52148600f806.png)

 and the code is to show the line around the tab, maybe that is the problem.

**Reply by black7375** [reply]
https://github.com/black7375/Firefox-UI-Fix/wiki/Tutorial#edit-userjs-file
```js
user_pref("userChrome.tab.box_shadow",                false);
```

You can set the above option to `false`, but there is no border.

---

There is a reason for this implementation.

At first, it was implemented as SVG's stroke, but it was very rough. #511

![image](https://user-images.githubusercontent.com/25581533/210427132-95083088-ef34-46bf-9af1-1d5ab40bf18f.png)

https://github.com/black7375/Firefox-UI-Fix/pull/511/commits/e10db8d9e7f1133bb7475b6e77dfcf72ff34c032

**Reply by Godiesc** [reply]
oooh, so the tab-line disappear jeje thanks for your help.

**Reply by black7375** [reply]
Rather than using rough curved borders across all themes, I decided to trade-off at the expense of compatibility in a transparent background theme.

**Reply by Godiesc** [reply]
I understand, make a complete compatibility with a lot of styles is very hard. Great job.
